network/dag/publisher.go

```diff
@@ -150,30 +150,27 @@ func (s *replayingDAGPublisher) publish(ctx context.Context) {
 	}
 }
 
-func (s *replayingDAGPublisher) handlePrivateTransaction(tx Transaction) bool {
+func (s *replayingDAGPublisher) handlePrivateTransaction(tx Transaction) {
 	_, err := s.privateTxCtx.PublishAsync(events.PrivateTransactionsSubject, tx.Data())
+
 	if err != nil {
 		log.Logger().Errorf("unable to handle private transaction: (ref=%s) %v", tx.Ref(), err)
-
-		return false
 	}
-
-	return true
 }
 
 func (s *replayingDAGPublisher) publishTransaction(ctx context.Context, transaction Transaction) bool {
-	// We need to skip transactions with PAL header as it should be handled by the v2 protocol
-	if len(transaction.PAL()) > 0 {
-		return s.handlePrivateTransaction(transaction)
-	}
-
 	payload, err := s.payloadStore.ReadPayload(ctx, transaction.PayloadHash())
 	if err != nil {
 		log.Logger().Errorf("Unable to read payload to publish DAG: (ref=%s) %v", transaction.Ref(), err)
 		return false
 	}
 
 	if payload == nil {
+		// We need to skip transactions with PAL header as it should be handled by the v2 protocol
```

Comment:
Nitpick: comment is not accurate anymore (`s/skip/process/` or something similar)